No content packages found for project dependencies when using type=content-package

When using dependencies of type `content-package` instead of type `zip` the convert mojo fails with

```
[ERROR] Failed to execute goal com.adobe.aem:aemanalyser-maven-plugin:0.0.16:convert (default-convert) on project aem-analyser-example: No content packages found for project. -> [Help 1]
```

Example project at https://github.com/kwin/aem-analyser-example/tree/type-content-package

--- a/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/ConvertToFeatureModelMojo.java
+++ b/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/ConvertToFeatureModelMojo.java
@@ -70,7 +70,7 @@ public class ConvertToFeatureModelMojo extends ConvertCPMojo {
         List<ContentPackage> l = new ArrayList<>();
 
         for (Dependency d : project.getDependencies()) {
-            if ("zip".equals(d.getType())) {
+            if ("zip".equals(d.getType()) || "content-package".equals(d.getType())) {
                 // If a dependency is of type 'zip' it is assumed to be a content package. TODO find a better way...
                 ContentPackage cp = new ContentPackage();
                 cp.setGroupId(d.getGroupId());

--- a/aemanalyser-maven-plugin/src/test/java/com/adobe/aem/analyser/mojos/ConvertToFeatureModelMojoTest.java
+++ b/aemanalyser-maven-plugin/src/test/java/com/adobe/aem/analyser/mojos/ConvertToFeatureModelMojoTest.java
@@ -61,6 +61,10 @@ public class ConvertToFeatureModelMojoTest {
         dep2.setType("pom");
         dep2.setGroupId("dg2");
         dep2.setArtifactId("da2");
+        Dependency dep3 = new Dependency();
+        dep2.setType("content-package");
+        dep2.setGroupId("dg3");
+        dep2.setArtifactId("da3");
 
         Build build = new Build();
         build.setDirectory(tempDir.toString());
@@ -87,10 +91,13 @@ public class ConvertToFeatureModelMojoTest {
         @SuppressWarnings("unchecked")
         List<ContentPackage> cpl = (List<ContentPackage>) TestUtil.getField(
                 mojo, "contentPackages");
-        assertEquals(1, cpl.size());
+        assertEquals(2, cpl.size());
         ContentPackage cp = cpl.get(0);
         assertEquals("dg", TestUtil.getField(cp, "groupId"));
         assertEquals("da", TestUtil.getField(cp, "artifactId"));
+        cp = cpl.get(1);
+        assertEquals("dg3", TestUtil.getField(cp, "groupId"));
+        assertEquals("da3", TestUtil.getField(cp, "artifactId"));
     }
 
     @Test

